### PR TITLE
Initial Recipe for Hill Range CKEditor integration for Symfony 4+

### PR DESCRIPTION
**Pull request by crayner**
| Q             | A
| ------------- | ---
| License       | MIT
| ------------- | ---
| Version       | 0.1.04

### CODE REVIEW

File: hillrange/ckeditor/0.1/config/packages/hillrange_ck_editor.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+                - ['Image','Table','-','Link','Flash','Smiley','TextColor','BGColor','Source']
+            uiColor: "ddcccc"
+        simple_toolbar:
+            toolbar:
+                - ['Bold', 'Italic', 'Strike', 'Link']
+                - ['BulletedList', 'NumberedList', '-', 'Outdent', 'Indent']
+                - ['Copy', 'Paste', 'PasteFromWord', '-', 'Undo', 'Redo']
+                - ['Source']
+twig:
+    paths:
```
Comment (Nyholm) [inline hillrange/ckeditor/0.1/config/packages/hillrange_ck_editor.yaml]: We should not configure other bundles.

**Comment by crayner** [comment]
Thanks for the detail.

Happy to remove the twig config, but I have NOT found any solution for to
work from the app to the bundle.  Will add the post install note.

Travis is red because it fails on Symfony 3.4.  As this is a Symfony 4+
bundle. this will not be fixed.  Most of the changes I needed to do where
in the composer.json of the bundle, and these changes do not refresh the
test results.

------------------------------------------
I greet you with the great words,
Grace and Peace

Craig Rayner
craig@craigrayner.com
Phone: +61 2 6352 5571
Mobile: +61 456 560 018

File: hillrange/ckeditor/0.1/manifest.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Hillrange\\CKEditor\\HillrangeCKEditorBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "composer-scripts": {
+        "cache:clear": "symfony-cmd",
```
Comment (Nyholm) [inline hillrange/ckeditor/0.1/manifest.json]: This is done by other bundle. Please remove.

File: hillrange/ckeditor/0.1/manifest.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+        "ckeditor:install": "symfony-cmd",
+        "assets:install --symlink --relative %PUBLIC_DIR%": "symfony-cmd"
```
Comment (Nyholm) [inline hillrange/ckeditor/0.1/manifest.json]: This is done by other bundle. Please remove. 

Make sure your bundle has a dependency on symfony/asset

Comment (crayner) [reply hillrange/ckeditor/0.1/manifest.json]: Thanks Tobias,  This has all been done in version 0.1.07.  The install of ckeditor remains as it is unique to this bundle.

Comment (Nyholm) [reply hillrange/ckeditor/0.1/manifest.json]: Im not sure what you mean. 
`assets:install` and `cache:clear` is not unique to this bundle. They should be removed from this recipe. 

The `ckeditor:install` command should remain.

**Comment by crayner** [comment]
oops., forgot to push the flex recipe...

------------------------------------------
I greet you with the great words,
Grace and Peace

Craig Rayner
craig@craigrayner.com
Phone: +61 2 6352 5571
Mobile: +61 456 560 018

File: hillrange/ckeditor/0.1/manifest.json [hunk 4]
```diff
@@ -6,8 +6,6 @@
         "config/": "%CONFIG_DIR%/"
     },
     "composer-scripts": {
-        "cache:clear": "symfony-cmd",
         "ckeditor:install": "symfony-cmd",
```
Comment (Nyholm) [inline hillrange/ckeditor/0.1/manifest.json]: Remove the comma.

Comment (crayner) [reply hillrange/ckeditor/0.1/manifest.json]: done

**Comment by crayner** [comment]
Thanks Tobias

------------------------------------------
I greet you with the great words,
Grace and Peace

Craig Rayner
craig@craigrayner.com
Phone: +61 2 6352 5571
Mobile: +61 456 560 018

**Pull request merged**